website/docs/reference/node-selection/syntax.md

```diff
@@ -61,3 +61,6 @@ $ dbt run --models tag:nightly my_model finance.base.*
 # use methods and intersections for more complex selectors
 $ dbt run --models path:marts/finance,tag:nightly,config.materialized:table
 ```
+
+:::info
+The shorthand of --models that can be used for selecting models is -m 
```

Comment:
I was referencing the documentation here during a lesson and noticed that the -m flag was not listed on this page.  Not sure where in the docs it is listed.  Definitely needs better phrasing here.

Participant was like, "where does is mention the `-m` flag?  That should be in the docs here" so I was like, "Let's do that right now." 😬

Reply:
Makes sense, but I think overall these docs _really_ need some love. Check out issue #244

Reply:
(Cleaning up my PRs in this repo) I'll drop this for handling this in future refactors addressing that issue.